articles/nodejs-use-node-modules-azure-apps.md

```diff
@@ -47,7 +47,7 @@ Azure App Service does not support all native modules, and might fail when compi
 
   * Before compiling, check that your local Node.js installation has matching architecture and the version is as close as possible to the one used in Azure (the current values can be checked on runtime from properties **process.arch** and **process.version**).
 
-* Azure App Service can be configured to execute custom bash or shell scripts during deployment, giving you the opportunity to execute custom commands and precisely configure the way **npm install** is being run. For a video showing how to configure that environment, see [Custom Website Deployment Scripts with Kudu].
+* Azure App Service can be configured to execute custom bash or shell scripts during deployment, giving you the opportunity to execute custom commands and precisely configure the way **npm install** is being run. For a video showing how to configure that environment, see [Custom Website Deployment Scripts with Kudu](https://azure.microsoft.com/en-us/resources/videos/custom-web-site-deployment-scripts-with-kudu/).
```

Comment:
Can you remove the hardcoded en-us region string from this URL?

Reply:
How can I add the change?